core/src/test/scala/unit/kafka/server/SaslApiVersionsRequestTest.scala

```diff
@@ -51,7 +51,6 @@ object SaslApiVersionsRequestTest {
 
     // Configure control plane listener to make sure we have separate listeners for testing.
     val serverProperties =  new java.util.HashMap[String, String]()
-    serverProperties.put(SocketServerConfigs.CONTROL_PLANE_LISTENER_NAME_CONFIG, controlPlaneListenerName)
     serverProperties.put(SocketServerConfigs.LISTENER_SECURITY_PROTOCOL_MAP_CONFIG, s"$controlPlaneListenerName:$securityProtocol,$securityProtocol:$securityProtocol")
```

Comment:
I think `controlPlaneListenerName` is unnecessary to this test, but this test is disabled for now. Maybe we can revisit it in KAFKA-17631

Reply:
oh, #18330 handles it already